packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -96,6 +113,11 @@ export class VSXExtensionsContribution extends AbstractViewContribution<VSXExten
             execute: () => this.installFromVSIX()
         });
 
+        commands.registerCommand({ id: VSXExtensionsCommands.INSTALL_ANOTHER_VERSION.id }, {
+            execute: async (extension: VSXExtension) => this.installAnotherVersion(extension),
+            isEnabled: (extension: VSXExtension) => !extension.builtin && !!extension.downloadUrl
```

Comment:
Maybe a small comment that checking for `downloadUrl` guarantees it was installed from the / a VSX registry.